game/src/gameCore/race/RaceGameFactory.ts

```diff
@@ -72,8 +61,11 @@ export default class RaceGameFactory {
 				}
 			}
 		}
-
-		return new RaceGrid(tiles, gridWidth, gridHeight, itemsState);
+		let raceGrid = new RaceGrid(tiles, gridWidth, gridHeight, []);
+		for (let i = 0; i < RACE_CST.CIRCUIT.NUMBER_OF_ITEMS; i++) {
+			raceGrid.generateNewItem([], isSinglePlayer);
```

Comment:
Bonne stratégie, mais il faudrait enlever les startingPositions lors de la génération d'items. Je ne crois pas qu'on veut que des items puissent être générés sur la ligne de départ en début de partie.

Reply:
Elles sont déjà enlevées parce qu'on fait appel à isAvailableForANewItem ;) !
`public isAvailableForANewItem(): boolean {
		return this.isWalkable && !this.isStartPosition && !this.isFinishLine && this.getItem() == undefined;
	}`